src/curl.cpp

```diff
@@ -4218,6 +4218,7 @@ bool S3fsCurl::UploadMultipartPostComplete()
     if (it == responseHeaders.end()) {
         return false;
     }
+    std::string etag = trim(it->second, "\"");
```

Comment:
I'm a little confused what this is trying to do -- it seems to unconditionally removed quotes.  But shouldn't s3fs always send the same ETag to the server that it previously returned to us?

Reply:
OK, let's peel off it from a string strictly enclosed in double quotes.
If the ETag is inappropriate, it will be detected as an error during the upload process anyway.
To do this, I will add one utility function for string manipulation.